src/scripts/setup_test_data/owner.txt

```diff
@@ -0,0 +1 @@
+3432808
```

Comment:
note: owner.txt is being set up in two separate ways right now -- both should use this file

Reply:
will fix in a follow-up PR.